cmd/uplink/cmd/setup.go

```diff
@@ -69,7 +70,13 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 		return err
 	}
 
-	var override map[string]interface{}
+	var (
+		encKeyFilepath = filepath.Join(setupDir, ".enc.key")
```

Comment:
Use `.encryption.key` or similar. Users probably don't know what `enc` means. (I would also change flags, but we don't have aliases in place for that).

Reply:
> Use .encryption.key or similar. Users probably don't know what enc means.

Before changing it, could we agree on this please, the name that I used is just to use one, actually I doubted if we wanted to use the leading `.` because that's not a hidden file in all the platforms.

> I would also change flags, but we don't have aliases in place for that

I don't understand what you're saying.

Reply:
The uplink flags still use `--enc.key` etc. but we can leave that for now.

Reply:
Egon is saying use full full work "encryption" not "enc".  Dot files are a little bit of a hassle in Windows under certain circumstances, but probably not enough to actively avoid them.

Reply:
> Egon is saying use full full work "encryption" not "enc"

I got that part

Reply:
> The uplink flags still use --enc.key etc. but we can leave that for now.

I understand that you meant the `enc.` should be `encryption`, if it's that, then yes it's in that way and I understood what you meant with the _alias_.